LkeServices/Blockchain/SiriusWalletsService.cs

```diff
@@ -53,23 +54,39 @@ public async Task CreateWalletsAsync(string clientId, bool waitForCreation)
             {
                 if (accountResponse.Body.Items.Count == 0)
                 {
-                    _log.WriteInfo(nameof(CreateWalletsAsync), info: "Creating wallets in sirius", context: new { clientId });
+                    string accountRequestId = $"{_brokerAccountId}_{clientId}_account";
+                    string userRequestId = $"{_brokerAccountId}_{clientId}_user";
```

Comment:
better to have only a client id here. Since users can be shared between the broker accounts theoretically